frontend/src/scenes/map-level.ts

```diff
@@ -0,0 +1,46 @@
+import Phaser from "phaser";
+import { SceneKeys } from "./scene-keys";
+import { mapWidth, mapHeight, blockSize, colors } from "../constants";
+import { MapLogicalGenerator } from "../map/map-logical-generation";
+
+const mapLogicalGenerator = new MapLogicalGenerator(mapWidth, mapHeight);
+const map = mapLogicalGenerator.run();
+
+export class MapLevel extends Phaser.Scene {
+  constructor() {
+    super(SceneKeys.MAP_LEVEL);
+  }
+
+  preload() {
+    this.load.image("blue", "assets/tree.png"); // in progress
+  }
+
+  create() {
+    const rows = map.length;
+    const columns = map[0]?.length || 0;
+
+    // Calculates the start position of the board
+    const startX = this.scale.width - columns * blockSize;
```

Comment:
Creo que es por esto que te queda abajo

Reply:
this.scale.width que es eso?

Reply:
ah tenes razon, es el ancho de la escena, tipo de la pantalla completa, nice catchhhh

Reply:
ahi lo dividi a la mitad